client/tools/mgr-push/mgr-push.changes

```diff
@@ -1,3 +1,4 @@
+- Allow rhnpush to handle deb packages
```

Comment:
We always put a line break after the line of the previous header. So it should be like:

```suggestion
- Allow rhnpush to handle deb packages

```